conda/recipes/libcucim/meta.yaml

```diff
@@ -39,22 +39,46 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - binutils
     - cmake >=3.23.1,!=3.25.0
     - make
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
     - yasm # [x86_64]
   host:
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - libcufile {{ cuda11_libcufile_host_version }}         # [linux64]
+    - libcufile-dev {{ cuda11_libcufile_host_version }}     # [linux64]
+    - libnvjpeg {{ cuda11_libnvjpeg_host_version }}
+    - libnvjpeg-dev {{ cuda11_libnvjpeg_host_version }}
+    {% else %}
+    - libcufile-dev     # [linux64]
+    - libnvjpeg-dev
+    - libnvjpeg-static
+    {% endif %}
     - jbig
     - libwebp-base {{ libwebp_base_version }}
+    - nvtx-c >=3.1.0
     - openslide
     - xz
     - zlib
     - zstd
   run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    - libcufile {{ cuda11_libcufile_run_version }}      # [linux64]
+    {% else %}
+    - cuda-nvrtc
+    - libcufile  # [linux64]
```

Comment:
Do we not get this from the run export of libcufile-dev?

Reply:
Just [followed `libcudf`'s `libcufile` usage]( https://github.com/rapidsai/cudf/blob/120fc29ed8b4d407e3ce99fd40a67d772edb58d5/conda/recipes/libcudf/meta.yaml#L93-L100 ) here. Do we want to change `libcudf` as well? FWIW this sounds like a reasonable change to me

cc @bdice (for awareness)

Reply:
Ah actually think we made that change in cuDF due to multiple outputs not always working with `run_exports`, which doesn't apply here. Will make this change as well

Reply:
Done in commit ( https://github.com/rapidsai/cucim/pull/572/commits/e2b8fe4fce25eb300c256eea0b14f3eb0fbc6485 )

Reply:
Right yeah in libcudf it's the run_exports/multiple outputs incompatibility issues AFAIR.